app/forms/steps/case/case_type_form.rb

```diff
@@ -0,0 +1,59 @@
+module Steps
+  module Case
+    class CaseTypeForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+
+      attribute :case_type
```

Comment:
Just noticed the attributes are missing their types 🤣 
In this case not a big deal as I believe it defaults to being `:string` type but better to add them just in case hehe.